src/applications/vaos/new-appointment/index.jsx

```diff
@@ -110,6 +113,13 @@ export function NewAppointment() {
           path={`${match.url}/how-to-schedule`}
           component={ScheduleCernerPage}
         />
+        {featureCCIteration &&
```

Comment:
I think this should be `featureCCIteration || hasResidentialAddress`? We want to show the provider selection page when we're doing the iteration or when the user has an address.

Then the condition for `CommunityCarePreferencesPage` would be the opposite.